plugins/modules/azure_rm_keyvaultsecret_info.py

```diff
@@ -115,7 +115,7 @@
             returned: always
             type: dict
             sample: {"delete": "on-exit"}
-        type:
+        content_type:
```

Comment:
I'm not sure if it's a good idea to change the response of info module, but this variable has a more realistic name with content_type than with type... Could someone give me your opinion? :)